dbms/src/Storages/System/StorageSystemBuildOptions.generated.cpp.in

```diff
@@ -34,7 +34,6 @@ const char * auto_config_build[]
     "SPLIT_BINARY", "@CLICKHOUSE_SPLIT_BINARY@",
     "USE_EMBEDDED_COMPILER", "@USE_EMBEDDED_COMPILER@",
     "USE_GLIBC_COMPATIBILITY", "@GLIBC_COMPATIBILITY@",
-    "USE_JEMALLOC", "@USE_JEMALLOC@",
```

Comment:
How we will check it in `system.build_options`?

Reply:
Will return it back for `ENABLE_JEMALLOC` variable.